kubeflow/training-operator/gpt-neox/04-finetune-workflow.yaml

```diff
@@ -54,6 +54,19 @@ spec:
     # Number of nodes used for training
     - name: trainer_nodes
       value: 2
+    # Type of GPU to use for training
+    - name: trainer_gpu
+      value: "A100_NVLINK_80GB"
+    # Wether to request infiniband on the training nodes
+    - name: use_ib
+      value: true
+    # The following training config parameters have been tuned for training on 2 A100_NVLINK_80GB nodes
+    # Batch size that will be sent to each GPU, total batch size = micro_batch_size * num_gpus
+    - name: micro_batch_size
+      value: 8
+    # Number of steps to accumulate the gradient before applying it to the model weights (Also known as GAS)
+    - name: gradient_accumulation_steps
+      value: 96
```

Comment:
@wbrown - With these new parameters you can set everything needed to do a run on A40's
```bash
argo submit 04-finetune-workflow.yaml \            
    -p run_name=finetune_gpt_neox \
    -p trainer_gpu=A40 \
    -p use_ib=false \
    -p micro_batch_size=4 \
    -p gradient_accumulation_steps=192 \
    --serviceaccount finetune
```